src/a09-dates/verifiers/handler.js

```diff
@@ -0,0 +1,35 @@
+import { getCurrentFormattedDate } from './index.verify.js'
+import { mockCalls } from '../../../verify/test.verify.js'
+import { assertCalls } from '../../../verify/assert.verify.js'
+
+await import('../test/index.test.js')
+
+// I'm creating this wrapper in order to be able to spy the process.exit
+process.on('exit', () => {
```

Comment:
I think I commented on this before, can we extract this in a utility module so we don't have to do this for all verifiers across all steps?

Reply:
Fixed in #15. As soon as it gets merged, I'll update it also here